scripts/anvil_tools/batch_upsert_entities_standard.py

```diff
@@ -55,6 +55,30 @@ def convert_string_to_list(input_string):
     return output_list
 
 
+def create_list_attr_operation(var_attribute_list_name):
+    """Return json request for a single operation to create an attribute of type array/list."""
+
+    return '{"op":"CreateAttributeValueList","attributeName":"' + var_attribute_list_name + '"},'
+
+
+def add_list_member_operation(var_attribute_list_name, var_attribute_list_member):
+    """Return json request for a single operation to add a list member to an attribute of type array/list."""
+
+    return '{"op":"AddListMember","attributeListName":"' + var_attribute_list_name + '", "newMember":"' + var_attribute_list_member + '"},'
+
+
+def create_non_array_attr_operation(var_attribute_name, var_attribute_value):
+    """Return json request for a single operation to create a non-array attribute."""
+
+    return '{"op":"AddUpdateAttribute","attributeName":"' + var_attribute_name + '", "addUpdateAttribute":"' + var_attribute_value + '"},'
```

Comment:
why not remove the comma here so you don't have to do it later? actually I'd remove the comma in all of these functions and _add_ the commas when you're making the composite list